lale/lib/sklearn/robust_scaler.py

```diff
@@ -63,7 +63,14 @@
                     "description": "If False, try to avoid a copy and do inplace scaling instead.",
                 },
             },
-        }
+        },
+        {
+            "description": "From /preprocessing/_data.py:RobustScaler:fit, Exception: raise ValueError(     'Cannot center sparse matrices: use `with_centering=False` instead. See docstring for motivation and alternatives.'     ) ",
```

Comment:
Could you please simplify the description to elide the "From ... raise ..." boiler-plate?